generator/uscore/metadata_extractor.rb

```diff
@@ -556,6 +549,10 @@ def set_first_search(sequence, params)
         sequence[:searches].delete(search)
         sequence[:searches].unshift(search)
       end
+
+      def capitalize_first_letter(str)
```

Comment:
We should get rid of this method and go back to using `String#upcase_first`, which we get from `ActiveSupport`.

Reply:
changes are reversed. Why do we have to use different pattern in Community?